tests/cypress/specs/eligibility-no-auth.cy.js

```diff
@@ -0,0 +1,30 @@
+const helpers = require("./helpers");
+
+const verifier_selection_url = "/eligibility";
+
+describe("Multiple Verifier, no AuthProvider: Verifier selection page spec", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    helpers.selectAgency();
+  });
+
+  it("User sees two radio buttons", () => {
+    cy.get("input:radio").should("have.length", 2);
+    cy.contains("Courtesy Card");
```

Comment:
Would this same type of `.contains()` work to simplify the [`selectCourtesyCard` helper](https://github.com/cal-itp/benefits/pull/1277/files#diff-88f268da70bf80004fffc1b39325c8693737f2269ff8a95b808ff78ffc15192eR20)? 

I guess I'm not clear on why the text "Courtesy Card" is used here, but the position of the radio button in the list is used in the other case?

Reply:
Unfortunately it's not quite that simple. Here, it's simply checking whether the text is on the page, whereas to actually select the radio button, it needs to find the right `<input>`. Given that `Courtesy Card` appears as only part of the text in a `<span>` in the `<label>`, the selection isn't quite as simple.

<img width="1624" alt="Screenshot 2023-02-16 at 5 17 16 PM" src="https://user-images.githubusercontent.com/86842/219501503-f156407b-000f-4264-8221-34467e57e655.png">